custom-checks/checkstyle/src/test/resources/checkstyle/thingHandlerSetLabelCheckTest/org.openhab.binding.extending.handler/ThingHandlerExtendingIncorrectClass.java

```diff
@@ -0,0 +1,18 @@
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.example.test.core.thing.binding.ExampleAbstractClass;
+import org.eclipse.smarthome.core.types.Command;
+
+public class ThingHandlerExtendingIncorrectClass extends ExampleAbstractClass{
```

Comment:
Please format all test file I especially noted the missing spaces in front of the {